### DISCUSSION TITLE
大佬，我想加入团队🤒

### DISCUSSION THREAD

**Discussion by pkc918** [ikun-svelte/ikun-ui#446] [General]
🫡

**Comment by baiwusanyu-c** [comment]
![image](https://github.com/ikun-svelte/ikun-ui/assets/32354856/67ec4080-2f97-4cd7-abbd-e4e77395e89e)
都过期两次了👀

**Reply by pkc918** [reply]
卧槽，我没看到

**Reply by pkc918** [reply]
sorry

**Reply by pkc918** [reply]
太抱歉了，白雾佬🫡
